codebuild/codebuild_apply_terraform/code_build_terraform.yml

```diff
@@ -14,48 +14,48 @@ phases:
       - REPO=$(pwd | awk -F '/' '{print $NF}') # For the artefact name
       - ACTIONS_REQUIRED="${CODEBUILD_SRC_DIR_actions_required}/actions_required.json"
       - if [ -z  "$CHECK_TRIGGER" ] || [ "$CHECK_TRIGGER" -eq 0 ]; then
-          echo "CHECK_TRIGGER not supplied, defaulting to true..";
-          TRIGGER_BUILD=true;
-        elif [ ! -z "$CHECK_TRIGGER" ] && [ "$CHECK_TRIGGER" -eq 1 ]; then
-       	  TRIGGER_BUILD=$(jq --arg ACTION_NAME "$ACTION_NAME" '.[] | select(.action ==$ACTION_NAME).required' $ACTIONS_REQUIRED);
+         echo "CHECK_TRIGGER not supplied..";
+         TRIGGER_BUILD=0;
+        elif [ ! -z "$CHECK_TRIGGER" ] || [ "$CHECK_TRIGGER" -eq 1 ]; then
+          TRIGGER_BUILD=$(jq --arg ACTION_NAME "$ACTION_NAME" '.[] | select(.action ==$ACTION_NAME).required' $ACTIONS_REQUIRED);
         fi
-      - if $TRIGGER_BUILD; then
+      - if [ -z "$TRIGGER_BUILD" ] || [ "$TRIGGER_BUILD" -eq 1 ]; then
           echo '[INFO] Validate Terraform...';
           cd $TERRAFORM_DIRECTORY;
           terraform init --backend=false;
           terraform validate;
         else
-          echo "trigger build set to false, not validating terraform"   ;
+          echo "trigger build set to false, not validating terraform"
```

Comment:
I seem to remember that in codepipeline, statements within if / else statements need to end with a semicolon